.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -38,7 +36,7 @@ jobs:
         # language=bash
         run: |
           python -m pip install --upgrade pip setuptools wheel
-          python setup.py develop
+          python -m pip install .
```

Comment:
Should that be install -e .?

that's more equivalent to develop

Reply:
Maybe? Ahaha this "install using pyproject.toml" is so new to me 😅

Reply:
Yes, `-e` would be the direct equivalent.